packages/react/src/themes/teams/components/SplitButton/splitButtonStyles.ts

```diff
@@ -0,0 +1,54 @@
+import { ICSSInJSStyle } from '../../../types'
+import getBorderFocusStyles from '../../getBorderFocusStyles'
+import SplitButton from '../../../../components/SplitButton/SplitButton'
+
+const splitButtonStyles = {
+  button: ({ variables: v }): ICSSInJSStyle => ({
+    border: 0,
+    padding: v.padding,
+
+    ':focus-visible': {
+      '::before': { border: 0 },
+      '::after': { border: 0 },
+    },
+    ':active': { backgroundColor: v.backgroundColorActive },
+  }),
+  root: ({ props: p, variables: v, theme: { siteVariables } }): ICSSInJSStyle => {
+    const { borderWidth } = siteVariables
+    const borderFocusStyles = getBorderFocusStyles({
+      siteVariables,
+    })
+
+    return {
+      border: `${borderWidth} solid ${v.borderColor}`,
+      borderRadius: v.borderRadius,
+      position: 'relative',
+      display: 'inline-block',
+
+      [`& .${SplitButton.slotClassNames.toggleButton}`]: {
+        borderRight: 0,
+        borderTop: 0,
+        borderBottom: 0,
+      },
+
+      ':focus-within': {
+        boxShadow: 'none',
+        ...(p.isFromKeyboard
```

Comment:
`:focus-visible`

Reply:
if `:focus` in the generated selector will break things, let's create a followup PR to handle such case in the felaFocusVisibleEnhancer.ts